R/adrs_onco.R

```diff
@@ -69,9 +70,12 @@
 #'     \item{ EOSSTT }{End of Study Status}
 #'     \item{ FRVDT }{Final Retrievel Visit Date}
 #'     \item{ DTHDT }{Date of Death}
-#'     \item{ DTHDTF }{undocumented field}
+#'     \item{ DTHDTF }{Date of Death Imputation Flag}
 #'     \item{ DTHADY }{Relative Day of Death}
 #'     \item{ LDDTHELD }{Elapsed Days from Last Dose to Death}
+#'     \item{ DTHCAUS }{undocumented field}
+#'     \item{ DTHDOM }{undocumented field}
+#'     \item{ DTHCGR1 }{undocumented field}
```

Comment:
These variables are missing their label. Should that update be made directly in the R/adrs_onco.R file, or the spec?

Reply:
Nice catch! the update should be made directly in the `inst/extdata/adams-specs.xlsx`

Reply:
updated